dali/server/daldap.cpp

```diff
@@ -164,19 +164,23 @@ class CDaliLdapConnection: implements IDaliLdapConnection, public CInterface
 
                 CDateTime now;
                 now.setNow();
-                if (now.compare(reqUTCTimestamp) < 0)//timestamp from the future?
+                if (now.compare(reqUTCTimestamp, false) < 0)//timestamp from the future?
                 {
-                    ERRLOG("LDAP: getPermissions(%s) scope=%s user=%s Request digital signature timestamp %s from the future",key?key:"NULL",obj?obj:"NULL",username.str(), requestTimestamp.str());
+                    StringBuffer localDaliTimeUTC;
+                    now.getString(localDaliTimeUTC, false);//get UTC timestamp
+                    ERRLOG("LDAP: getPermissions(%s) scope=%s user=%s Request digital signature UTC timestamp %s from the future (Dali UTC time %s)",key?key:"NULL",obj?obj:"NULL",username.str(), requestTimestamp.str(), localDaliTimeUTC.str());
```

Comment:
Is there any worthwhile guidance we can provide to the admin reading the log regarding these types of errors?
-- confirm clock settings on XYZ machine, etc?

Reply:
Perhaps, but it should be obvious !